end_to_end_tests/golden-record-custom/custom_e2e/api/tests/upload_file_tests_upload_post.py

```diff
@@ -24,6 +24,8 @@ def _parse_response(*, response: httpx.Response) -> Optional[Union[
     if response.status_code == 422:
         response_422 = HTTPValidationError.from_dict(response.json())
 
+
+
```

Comment:
Odd, I'd think that black would have removed that.

Reply:
Yeah, I couldn't figure out why that got added by my PR...